web/viewer-geckoview.js

```diff
@@ -68,6 +68,7 @@ function webViewerLoad() {
 
       document.head.append(link);
     }
+    window.isGECKOVIEW = true;
```

Comment:
nit: use https://searchfox.org/mozilla-central/rev/08362489086b10de96e7a199b267ea5504c01583/toolkit/components/pdfjs/content/PdfStreamConverter.jsm#395

Reply:
Unfortunately that doesn't help in local development, i.e. when running `gulp server` and then opening http://localhost:8888/web/viewer-geckoview.html, which is what this patch tries to improve.

Reply:
Ah yes and I thought about that when I read it the first time...